### ISSUE TITLE
NEI show multiple gas and fluid tanks and lag ???

### ISSUE THREAD

**Issue by mcdodge34** [aidancbrady/Mekanism#3357]
Ok, my only complain for now is that NEI does show all the tanks with all the liquid and gas they can contain, which makes NEI lags like crazy while searching for stuff, is there any options we could get not to show them in NEI, only the empty ones and their crafting recipes should be enough, its like the Forge Multipart, in the past it was lagging NEI like hell until they decided to hide their recipes for all the blocks we could get. Using Mekanism V9.0.2.271 and also tested the 273 and still same problem, we can't filter this in NEI in the top since they are integrated into the mod itself, really a problem for someone that just want to learn how to use the mod.

**Comment by HadesDurin** [comment]
as stated https://github.com/aidancbrady/Mekanism/issues/3352 there is a fix in the mekanism config to prevent that

**Comment by mcdodge34** [comment]
I wasn't aware of theses options, thanks for pointing me to this thread, I'll try it, do I need to also disable them on server side or only on client will do, just want to avoid a server restart if possible, but if needed I can restart it with the correct configs.

**Comment by HadesDurin** [comment]
look closely at the points and find then
in your config they set to false, what is the option to disable them in nei

**Comment by mcdodge34** [comment]
I've linked my file no mention of any kind of portable tank there to be found.

For NEI, its a thing with clicking the menus on top of the screen, I don't remember exactly I think we could left or right click an option on a mod category to totally hide it from the NEI system, but since Mekanism only has 1 category, if we hide it, we won't see anything from Mekanism in our search at all.

**Comment by HadesDurin** [comment]
the option is 
B:PrefilledFluidTanks=false
which you have in your config file
and having it to false removes the prefilled tanks from the nei.
additionally if you missed it there are 4 tiers of tanks aviable from basic to ultimate
same for gas tanks

**Comment by mcdodge34** [comment]
Yup, I did it already I was confused cause I was sure there was a 3rd type of it, didn't know it was the portable one, I suppose its similar to the ones from TE that are pretty handy sometimes, even if I always prefer to use drums and bedrockium drums since they cause less lag. We can consider this issue closed for now.

**Comment by HadesDurin** [comment]
B:PrefilledFluidTanks=false
    B:PrefilledGasTanks=false
    B:PrefilledPortableTanks=false
there all three options you are searching for and if you look inside your config file you will see it

**Comment by mcdodge34** [comment]
Not sure why, but the 3rd line regarding the portabletanks isn't in my file anywhere, you can take a look at my config mekanism.cfg here : http://pastebin.com/Y40Pcfsq

Anyways, just disabling the two first ones solved the issue, I didn't even had to modify it on the server, will probably do it when I do a mod update to the pack at the end of the week.
